spec/controllers/openid_connect/logout_controller_spec.rb

```diff
@@ -65,6 +65,12 @@
               ),
             )
 
+          expect_any_instance_of(IrsAttemptsApi::Tracker).to receive(:logout_initiated).
```

Comment:
we may want to add a `stub_attempts_api` helper method like we have for `stub_analytics`

Reply:
Hey @zachmargolis, we'll add a story to the backlog and prioritize this work separately.  Thanks!

Reply:
To follow up, said story is at https://cm-jira.usa.gov/browse/LG-7133